handlers/strike.py

```diff
@@ -0,0 +1,43 @@
+import os
+from jsonpath_rw import jsonpath, parse
+
+class Strike():
+	def handleCommand(self, command):
+	
+		if command.command == "getStrikes"
+			return getStrikesString(self)
```

Comment:
we use 2 spaces for an indent level in this proj... mind doing ctrl-h find replace?

Reply:
notepad++ :(

Reply:
done